content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_completed.rb

```diff
@@ -20,33 +20,45 @@ def main
 
               private
 
-              def task
+              def update_task(message)
+                return unless service_action == 'Provision'
                 @handle.root["service_template_provision_task"].tap do |task|
```

Comment:
@tinaafitz Since task is optional could we use try
```
@handle.root['service_template_provision_task'].try { |task| task.miq_request.user_message = message }
```